hibernate-orm-quickstart/pom.xml

```diff
@@ -41,11 +41,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-jackson</artifactId>
+            <artifactId>quarkus-resteasy-reactive-jackson</artifactId>
```

Comment:
This is unrelated to the title of the PR... and while I think we should do it, we should probably do it all around the quickstarts if you and @geoand think it's a good idea to move the quickstarts to RESTEasy Reactive for 2.3.

I will be happy to do it as it will be a good opportunity to learn a bit more about it.

Reply:
Yeah, I think @cescoffier had a plan to do that for all of them